code/game/objects/items/devices/taperecorder.dm

```diff
@@ -138,12 +157,10 @@
 		return
 
 	if(mytape.used_capacity < mytape.max_capacity)
-		to_chat(usr, "<span class='notice'>Recording started.</span>")
 		playsound(src, 'sound/items/taperecorder/taperecorder_play.ogg', 50, FALSE)
-		recording = 1
+		recording = TRUE
+		say("<font color='[say_color]'>Recording started.</font>")
 		update_icon()
-		mytape.timestamp += mytape.used_capacity
-		mytape.storedinfo += "\[[time2text(mytape.used_capacity * 10,"mm:ss")]\] Recording started."
```

Comment:
Is this behavior gone?

Reply:
It says in the PR body that this is replaced by it recording itself saying it rather than writing it directly.